README.md

```diff
@@ -55,34 +53,35 @@ java -Dloader.path="PATH-TO-ADDITIONAL-PLUGINGS/custom-plugins.jar" \
 ```
 
 The plugin library can be swapped with any third party access-checker as
-described in the [plugins](plugins) directory and
-[the wiki](https://github.com/google/fhir-gateway/wiki/Understanding-access-checker-plugins).
+described in the [plugins](plugins) directory. 
 
-Note: Spring Boot is not a requirement for using FHIR Information Gateway; we
-just use it to simplify the
+**Note** Spring Boot is not a requirement for using the access-proxy; we just use it to simplify the
 [MainApp](exec/src/main/java/com/google/fhir/gateway/MainApp.java). The only
 Spring-related requirement is to do a
 [@ComponentScan](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/annotation/ComponentScan.html)
 to find all access-checker plugins in the classpath.
 
-# Configuration parameters
+# FHIR Info Gateway configuration parameters
 
-The configuration parameters are provided through environment variables:
+The FHIR Info Gateway configuration parameters are currently provided through environment
+variables:
 
-- `PROXY_TO`: The base url of the FHIR store e.g.:
+- **FHIR store location**: This is set by `PROXY_TO` environment variable, using
```

Comment:
I believe @williamito just recently changed these in #83; is there a particular reason you are reverting these changes?